postgres/codenamemap.yaml

```diff
@@ -14,12 +17,13 @@
   #}
 
   {# use upstream version if configured #}
-  {% set version = upstream_version|default(version) %}
+  {% if use_upstream_repo %}
+    {% set version = upstream_version %}
+  {% endif %}
 
 {{ codename|default(name, true) }}:
   # PostgreSQL packages are mostly downloaded from `main` repo component
-  pkg_repo: 'deb http://apt.postgresql.org/pub/repos/apt/ {{ name }}-pgdg main {{ version }}'
-  pkg_repo_humanname: PostgreSQL Official Repository
+  pkg_repo: 'deb http://apt.postgresql.org/pub/repos/apt {{ name }}-pgdg main {{ upstream_version }}'
```

Comment:
I'd leave `{{ version }}` here, as it's clearly being set  in the preceeding lines you added, and is more consistent.